src/orderedmap.h

```diff
@@ -334,6 +342,15 @@ OrderedMap<Key, Value>::OrderedMap(const OrderedMap<Key, Value>& other)
     copy(other);
 }
 
+#ifdef Q_COMPILER_INITIALIZER_LISTS
+template<typename Key, typename Value>
+OrderedMap<Key, Value>::OrderedMap(std::initializer_list<std::pair<Key, Value> > list)
+{
+    for (typename std::initializer_list<std::pair<Key,Value> >::const_iterator it = list.begin(); it != list.end(); ++it)
```

Comment:
I hate long lines! :P Can we create a typedef for the initializer list iterator and shorten the line length?